internal/yunikorn/rest.go

```diff
@@ -29,6 +30,15 @@ type RESTClient struct {
 	port     int
 }
 
+type AggregatedStateInfo struct {
```

Comment:
Why not use yunikorn-core `webservice.AggregatedStateInfo`? We depend already on yunikorn-core, so why not simply re-use their structure that is public?